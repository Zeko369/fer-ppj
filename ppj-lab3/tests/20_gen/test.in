<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 x
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       OP_PLUS 1 +
       <P>
        BROJ 1 1064
      <T_lista>
       OP_PUTA 1 *
       <T>
        <P>
         L_ZAGRADA 1 (
         <E>
          <T>
           <P>
            BROJ 1 439
           <T_lista>
            $
          <E_lista>
           $
         D_ZAGRADA 1 )
        <T_lista>
         OP_PUTA 1 *
         <T>
          <P>
           BROJ 1 996
          <T_lista>
           $
     <E_lista>
      OP_PLUS 1 +
      <E>
       <T>
        <P>
         L_ZAGRADA 1 (
         <E>
          <T>
           <P>
            BROJ 1 778
           <T_lista>
            $
          <E_lista>
           $
         D_ZAGRADA 1 )
        <T_lista>
         OP_PUTA 1 *
         <T>
          <P>
           BROJ 1 1043
          <T_lista>
           $
       <E_lista>
        OP_MINUS 1 -
        <E>
         <T>
          <P>
           BROJ 1 720
          <T_lista>
           OP_PUTA 1 *
           <T>
            <P>
             BROJ 1 285
            <T_lista>
             $
         <E_lista>
          OP_PLUS 1 +
          <E>
           <T>
            <P>
             OP_MINUS 1 -
             <P>
              BROJ 1 910
            <T_lista>
             $
           <E_lista>
            $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 x
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        L_ZAGRADA 2 (
        <E>
         <T>
          <P>
           BROJ 2 1126
          <T_lista>
           $
         <E_lista>
          OP_PLUS 2 +
          <E>
           <T>
            <P>
             BROJ 2 587
            <T_lista>
             $
           <E_lista>
            $
        D_ZAGRADA 2 )
       <T_lista>
        OP_PUTA 2 *
        <T>
         <P>
          BROJ 2 627
         <T_lista>
          OP_PUTA 2 *
          <T>
           <P>
            BROJ 2 1133
           <T_lista>
            $
      <E_lista>
       OP_PLUS 2 +
       <E>
        <T>
         <P>
          L_ZAGRADA 2 (
          <E>
           <T>
            <P>
             L_ZAGRADA 2 (
             <E>
              <T>
               <P>
                BROJ 2 1023
               <T_lista>
                $
              <E_lista>
               $
             D_ZAGRADA 2 )
            <T_lista>
             $
           <E_lista>
            OP_PLUS 2 +
            <E>
             <T>
              <P>
               IDN 2 x
              <T_lista>
               OP_PUTA 2 *
               <T>
                <P>
                 OP_MINUS 2 -
                 <P>
                  BROJ 2 956
                <T_lista>
                 $
             <E_lista>
              $
          D_ZAGRADA 2 )
         <T_lista>
          $
        <E_lista>
         $
   <lista_naredbi>
    <naredba>
     <naredba_pridruzivanja>
      IDN 3 x
      OP_PRIDRUZI 3 =
      <E>
       <T>
        <P>
         L_ZAGRADA 3 (
         <E>
          <T>
           <P>
            BROJ 3 333
           <T_lista>
            $
          <E_lista>
           OP_PLUS 3 +
           <E>
            <T>
             <P>
              BROJ 3 1027
             <T_lista>
              $
            <E_lista>
             OP_MINUS 3 -
             <E>
              <T>
               <P>
                BROJ 3 654
               <T_lista>
                $
              <E_lista>
               $
         D_ZAGRADA 3 )
        <T_lista>
         $
       <E_lista>
        $
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 4 x
       OP_PRIDRUZI 4 =
       <E>
        <T>
         <P>
          OP_MINUS 4 -
          <P>
           BROJ 4 45
         <T_lista>
          $
        <E_lista>
         OP_PLUS 4 +
         <E>
          <T>
           <P>
            IDN 4 x
           <T_lista>
            $
          <E_lista>
           OP_MINUS 4 -
           <E>
            <T>
             <P>
              BROJ 4 797
             <T_lista>
              OP_PUTA 4 *
              <T>
               <P>
                OP_PLUS 4 +
                <P>
                 BROJ 4 244
               <T_lista>
                $
            <E_lista>
             $
     <lista_naredbi>
      <naredba>
       <naredba_pridruzivanja>
        IDN 5 x
        OP_PRIDRUZI 5 =
        <E>
         <T>
          <P>
           BROJ 5 320
          <T_lista>
           $
         <E_lista>
          OP_MINUS 5 -
          <E>
           <T>
            <P>
             BROJ 5 1095
            <T_lista>
             $
           <E_lista>
            $
      <lista_naredbi>
       $
