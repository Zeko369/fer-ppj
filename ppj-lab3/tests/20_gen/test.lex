IDN 1 x
OP_PRIDRUZI 1 =
OP_PLUS 1 +
BROJ 1 1064
OP_PUTA 1 *
L_ZAGRADA 1 (
BROJ 1 439
D_ZAGRADA 1 )
OP_PUTA 1 *
BROJ 1 996
OP_PLUS 1 +
L_ZAGRADA 1 (
BROJ 1 778
D_ZAGRADA 1 )
OP_PUTA 1 *
BROJ 1 1043
OP_MINUS 1 -
BROJ 1 720
OP_PUTA 1 *
BROJ 1 285
OP_PLUS 1 +
OP_MINUS 1 -
BROJ 1 910
IDN 2 x
OP_PRIDRUZI 2 =
L_ZAGRADA 2 (
BROJ 2 1126
OP_PLUS 2 +
BROJ 2 587
D_ZAGRADA 2 )
OP_PUTA 2 *
BROJ 2 627
OP_PUTA 2 *
BROJ 2 1133
OP_PLUS 2 +
L_ZAGRADA 2 (
L_ZAGRADA 2 (
BROJ 2 1023
D_ZAGRADA 2 )
OP_PLUS 2 +
IDN 2 x
OP_PUTA 2 *
OP_MINUS 2 -
BROJ 2 956
D_ZAGRADA 2 )
IDN 3 x
OP_PRIDRUZI 3 =
L_ZAGRADA 3 (
BROJ 3 333
OP_PLUS 3 +
BROJ 3 1027
OP_MINUS 3 -
BROJ 3 654
D_ZAGRADA 3 )
IDN 4 x
OP_PRIDRUZI 4 =
OP_MINUS 4 -
BROJ 4 45
OP_PLUS 4 +
IDN 4 x
OP_MINUS 4 -
BROJ 4 797
OP_PUTA 4 *
OP_PLUS 4 +
BROJ 4 244
IDN 5 x
OP_PRIDRUZI 5 =
BROJ 5 320
OP_MINUS 5 -
BROJ 5 1095
