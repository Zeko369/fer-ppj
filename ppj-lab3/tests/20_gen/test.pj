x = +1064*(439)*996 + (778)*1043 - 720*285 + -910
x = (1126 + 587)*627*1133 + ((1023) + x*-956)
x = (333 + 1027 - 654)
x = -45 + x - 797*+244
x = 320 - 1095
