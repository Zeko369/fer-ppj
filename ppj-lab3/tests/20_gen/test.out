2 1 x
4 1 x
